integration-tests/contract-manager-that-calls-failwith-contract.spec.ts

```diff
@@ -1,37 +1,44 @@
 import { CONFIGS } from "./config";
 import { failwithContractCode } from "./data/failwith";
 import { managerCode } from "./data/manager_code";
-import { MANAGER_LAMBDA } from "@taquito/taquito";
+import { DefaultContractType, MANAGER_LAMBDA, OriginationOperation } from "@taquito/taquito";
 
 CONFIGS().forEach(({ lib, rpc, setup }) => {
   const Tezos = lib;
 
-  describe(`Test contract origination of a contract that calls 2nd contract that FAILs through contract api: ${rpc}`, () => {
+  let contract: DefaultContractType;
```

Comment:
Also for future, when some data is needed in a test, we can create a function that's called in the `it` and returns the needed data. It's functionally not important, as tests inside one file are run in series and not in parallel,